src/package.xml

```diff
@@ -1664,6 +1679,12 @@
         <members>Contact-iw</members>
         <members>Contact-ja</members>
         <members>Contact-nl_NL</members>
+        <members>DataImportBatch__c-de</members>
```

Comment:
The consensus is to only update them when we have new translations to update. However, we are still trying to iron out the details for ignoring the changes but not removing them from the package. I wouldn't add any new English translation files though, I'm still hoping we can eliminate those from the package at some point.